app/controllers/admin/enterprises_controller.rb

```diff
@@ -142,7 +142,15 @@ def collection
         @order_cycle = OrderCycle.find_by_id(params[:order_cycle_id]) if params[:order_cycle_id]
         coordinator = Enterprise.find_by_id(params[:coordinator_id]) if params[:coordinator_id]
         @order_cycle = OrderCycle.new(coordinator: coordinator) if @order_cycle.nil? && coordinator.present?
-        OpenFoodNetwork::OrderCyclePermissions.new(spree_current_user, @order_cycle).visible_enterprises
+
+        enterprises = OpenFoodNetwork::OrderCyclePermissions.new(spree_current_user, @order_cycle)
+          .visible_enterprises
+
+        unless enterprises.empty?
```

Comment:
why can't we just concat this with the `includes` below. Isn't it an ActiveRecord::Relation? It looks like it is by looking at line 166.

Reply:
There were lots of spec failures when I tried that. In some of the tests it's an empty array.